### PR TITLE
add export keyword

### PR DESCRIPTION
**Pull request by Victor-Su-Ortiz**
add export keyword

While working on setting up aiconfig, there was a bit of confusion surrounding the OPENAI_API_KEY. Thought we should be explicit in including the export keyword in all of the comments.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "aiconfig": "^1.1.1"
+  }
```
Comment (rossdanlm) [inline package.json:4]: I think most of our cookbooks now are in python so can we remove? Can you share the documentation you followed so we can learn how you got here? Thanks!

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1606 @@
+{
+  "name": "aiconfig",
```
Comment (rossdanlm) [inline package-lock.json:2]: Can we uncommit this file from this?

File: cookbooks/Function-Calling-OpenAI/python/Function_Calling_with_OpenAI.ipynb [hunk 3]
```diff
@@ -23,7 +23,7 @@
         "!pip install python-aiconfig\n",
         "\n",
         "# Create .env file at cookbooks/Function-Calling-OpenAI/python/.env containing the \n",
```
Comment (rossdanlm) [inline cookbooks/Function-Calling-OpenAI/python/Function_Calling_with_OpenAI.ipynb:25]: I synced w. @rholinshead  and actually it turns out you CAN just do `OPENAI_API_KEY=<your key here>` without EXPORT. What I want to try is changing the `.env` file to be under `aiconfig/` dir, so could you change this line to this?

```suggestion
"# Create .env file at aiconfig/.env containing the following line:\n"
```

File: cookbooks/Getting-Started/getting_started.ipynb [hunk 4]
```diff
@@ -21,7 +21,7 @@
         "!pip install python-aiconfig\n",
         "\n",
         "# Create .env file at cookbooks/Function-Calling-OpenAI/python/.env containing the \n",
-        "# following line: OPENAI_API_KEY=<your key here>\n",
+        "# following line: export OPENAI_API_KEY=<your key here>\n",
```
Comment (rossdanlm) [inline cookbooks/Getting-Started/getting_started.ipynb:24]: Same as last comment

**Review by rossdanlm — COMMENTED** [review]
Looks great, pls update with comments, thanks for this and congrats on your first open source PR!

File: package.json (hunk 1 above)
Comment (Victor-Su-Ortiz) [reply package.json:4]: I was following the readme here to get started https://github.com/lastmile-ai/aiconfig

**Review by rossdanlm — CHANGES_REQUESTED** [review]
Just going to put back into queue for now so others are aware and don't review until ready :)

**Review by rossdanlm — APPROVED** [review]
Awesome thanks!!!

**Comment by saqadri** [comment]
Is this diff ready to be merged?

**Comment by rossdanlm** [comment]
> Is this diff ready to be merged?

Yes it is, this is Victor's first OS pull request, so he didn't realize he needed to click "Merge pull request". Synced with him offline :)

**Pull request merged**